dspace-api/src/main/java/org/dspace/sort/OrderFormatAuthor.java

```diff
@@ -21,6 +22,7 @@
 {
 	{
 		filters = new TextFilter[] { new DecomposeDiactritics(),
+				                     new StripDiacritics(),
```

Comment:
I could be wrong, but do we need *both* `DecomposeDiacritics()` and `StripDiacritics()` together?  Based on their descriptions, it seems like we should be using one or the other...but, maybe I'm misunderstanding this change?

Reply:
That's what I thought, too. But I can confirm that both are needed. StripDiacritics() needs decomposed diacritics to properly remove them.

This is explained in this old JIRA comment https://jira.duraspace.org/browse/DS-916?focusedCommentId=20559&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20559

Reply:
@toniprieto : Thanks for that extra info! I wasn't aware of that.  This looks good then & I'll resolve both my comments.